Tasks/JavaToolInstallerV0/taskutils.ts

```diff
@@ -0,0 +1,57 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+import * as os from 'os';
+
+import { ToolRunner } from 'azure-pipelines-task-lib/toolrunner';
+
+/**
+ * Returns promise which will be resolved in given number of milliseconds.
+ * @param sleepDurationInMilliSeconds Number of milliseconds.
+ * @returns Promise<any>
+ */
+export function sleepFor(sleepDurationInMilliSeconds: number): Promise<any> {    
+    return new Promise((resolve, reject) => {
+        setTimeout(resolve, sleepDurationInMilliSeconds);
+});
```

Comment:
Nit: could you please apply auto-formatting here?

Reply:
Thanks, applied